scripts/generator/graphql_generator/csharp/SDK/iOS/iOSNativeCheckout.ApplePayEventReceiver.cs.erb

```diff
@@ -60,7 +60,76 @@ namespace <%= namespace %>.SDK.iOS {
         }
 
         public void FetchApplePayCheckoutStatusForToken(string serializedMessage) {
```

Comment:
I suppose reading this from the `Swift` end it might make more sense, to me this method is similar to an HTTP request. I'm not sure which keyword we could use to make this sound more clear, `Get`?.